public/app/features/dashboard/components/SubMenu/DashboardLinksDashboard.tsx

```diff
@@ -79,6 +81,16 @@ export class DashboardLinksDashboard extends PureComponent<Props, State> {
     );
   };
 
+  getDropdownLocation = (): string => {
+    const [leftPosition, rightPosition] = ['pull-left', 'pull-right'];
+    const wrapper = this.wrapperRef.current;
+    const list = this.listItemRef.current;
+    if (!wrapper || !list) {
+      return rightPosition;
+    }
+    return wrapper.offsetLeft > list.offsetWidth - wrapper.offsetWidth ? rightPosition : leftPosition;
+  };
+
```

Comment:
nitpick: maybe the naming could be slightly improved with `getDropdownPosition` as variables inside the func are called `leftPosition, rightPosition`. wdyt?

Reply:
Maybe it would improve naming if I change function name to `getDropdownPositionCssClass` and change names of the variables to `pullLeftCssClass` and `pullRightCssClass`